packages/design-system/src/components/Dropdown/Dropdown.tsx

```diff
@@ -2,6 +2,7 @@ import React, { useMemo, useRef } from 'react';
 import classNames from 'classnames';
 import useAutofocus from '../utilities/useAutoFocus';
 import { FormFieldProps, FormLabel, useFormLabel } from '../FormLabel';
+import { ArrowIcon, CheckIcon } from '../Icons';
```

Comment:
These icons might be placeholders for now... We don't have the icons from the designs yet.